filter_linux.go

```diff
@@ -446,6 +450,10 @@ func (h *Handle) FilterList(link Link, parent uint32) ([]Filter, error) {
 				default:
 					detailed = true
 				}
+			case nl.TCA_CHAIN:
+				val := new(uint32)
+				*val = binary.LittleEndian.Uint32(attr.Value)
```

Comment:
system may be big-endian (e.g there are powerpc cpus that are big endian)

you can just use `netlink.native.Uint32()` like is being used in other places.